utils/Makefile

```diff
@@ -93,7 +98,11 @@ ci-flake8:
 	@echo "# Running Flake8 Compliance Tests in virtualenv"
 	@echo "#############################################"
 	. $(NAME)env/bin/activate && flake8 --config=setup.cfg ../ --exclude="utils,../inventory"
-	. $(NAME)env/bin/activate && python setup.py flake8 
+	. $(NAME)env/bin/activate && python setup.py flake8
 
-ci: clean virtualenv ci-list-deps ci-flake8 ci-pylint ci-unittests
-	:
+ci: venv ci-list-deps ci-unittests ci-flake8 ci-pylint
+	@echo
+	@echo "##################################################################################"
+	@echo "VIEW CODE COVERAGE REPORT WITH 'xdg-open cover/index.html' or run 'make viewcover'"
+	@echo "To clean your test environment run 'make clean'"
+	@echo "Other targets you may run with 'make': 'ci-pylint', 'ci-unittests', 'ci-flake8'"
```

Comment:
@rhcarvalho the whole reason the 'make ci' system was added was to enable contributors to run the tests on their local machines before having to wait on jenkins results to report in. I.e., so you only have to git push commits that you know pass the basic essential tests. That's why you can run them all individually and why the information is printed out with such verbosity. To make it easier on **people** using the ci system.